### ISSUE TITLE
Known Bugs

### ISSUE THREAD

**Issue by sherylll** [Infineon/platformio-infineonxmc#2]
In VS Code:

- For versions v1.0.1 and newer the Project Wizard from PIO home returns following error:
```
PIO Core Call Error: No JSON object could be decoded
```
However it is still possible to create a project manually and compile/upload it.

- Travis currently stalled on 
```
platformio platform install -f file://.
```

- Linux support to be offered

**Comment by ivankravets** [comment]
Fixed in https://github.com/Infineon/platformio-infineonxmc/commit/a753332f75563ab4bfffacd4fd22d80ae8370b77

**Comment by ivankravets** [comment]
@sherylll Could you fix https://travis-ci.org/Infineon/platformio-infineonxmc/jobs/446234219 ?

**Comment by sherylll** [comment]
fixed by 23258498226c1e0f9cacddcbf803add357642719

**Comment by ivankravets** [comment]
Thank you so much!
